smartconfig/_registry.py

```diff
@@ -66,7 +65,7 @@ def unload_defaults(path: str) -> None:
         raise ConfigurationError(f"Node at path {path!r} isn't a mapping.")
 
     for attribute in copy.copy(node):
-        if attribute not in overwritten_attributes:
+        if f"{path}.{attribute}" not in overwritten_attributes:
```

Comment:
What if the path is an empty string i.e. the root? Is that even a possible case?

Reply:
It shouldn't be possible afaik as the root node would just be the entry itself.

Reply:
But can't someone set the path of a `ConfigEntry` to be the root node?

Reply:
Nope, it will default to the module name if an empty string is passed.